cmd/convox/templates/init/rails/config/initializers/convox.rb

```diff
@@ -0,0 +1,5 @@
+STDOUT.sync = true
```

Comment:
When you volume mount the app directory the file from the image becomes unavailable because it isn't present on the host.